src/ensembl/src/content/app/entity-viewer/gene-view/components/default-transcripts-list/DefaultTranscriptsList.tsx

```diff
@@ -14,8 +19,54 @@ type Props = {
   rulerTicks: TicksAndScale;
 };
 
+const compareTranscriptLength = (
+  transcriptOne: Transcript,
+  transcriptTwo: Transcript
+) => {
+  const transcriptOneLength = getFeatureLength(transcriptOne);
+  const transcriptTwoLength = getFeatureLength(transcriptTwo);
+
+  if (transcriptOneLength < transcriptTwoLength) {
+    return -1;
+  }
+
+  if (transcriptOneLength > transcriptTwoLength) {
+    return 1;
+  }
+
+  return 0;
+};
+
+const sortTranscripts = (transcripts: Transcript[]) => {
```

Comment:
According to design, there will be different sorting options. Perhaps this calls for moving the sorting code in a separate file (`sortTranscripts.ts`? `transcriptsSorter.ts`? don't know what name will be best), and calling this sorting function `defaultSort` (it won't even need to have the word "transcripts" in its name)?

Reply:
✔️